---
title: "The MiST2 database: a comprehensive genomics resource on microbial
  signal transduction"
publication_types:
  - "2"
authors:
  - Luke E. Ulrich
  - Igor B. Zhulin
doi: https://doi.org/10.1093/nar/gkp940
publication_short: "Nucleic Acids Research 38: D401-D407"
abstract: "The MiST2 database (http://mistdb.com) identifies and catalogs the
  repertoire of signal transduction proteins in microbial genomes. Signal
  transduction systems regulate the majority of cellular activities including
  the metabolism, development, host-recognition, biofilm production, virulence,
  and antibiotic resistance of human pathogens. Thus, knowledge of the proteins
  and interactions that comprise these communication networks is an essential
  component to furthering biomedical discovery. These are identified by
  searching protein sequences for specific domain profiles that implicate a
  protein in signal transduction. Compared to the previous version of the
  database, MiST2 contains a host of new features and improvements including the
  following: draft genomes; extracytoplasmic function (ECF) sigma factor protein
  identification; enhanced classification of signaling proteins; novel,
  high-quality domain models for identifying histidine kinases and response
  regulators; neighboring two-component genes; gene cart; better search
  capabilities; enhanced taxonomy browser; advanced genome browser; and a
  modern, biologist-friendly web interface. MiST2 currently contains 966
  complete and 157 draft bacterial and archaeal genomes, which collectively
  contain more than 245 000 signal transduction proteins. The majority (66%) of
  these are one-component systems, followed by two-component proteins (26%),
  chemotaxis (6%), and finally ECF factors (2%)."
draft: false
featured: false
image:
  filename: featured
  focal_point: Smart
  preview_only: false
date: 2009-11-08T04:32:46.298Z
---
